cookbooks/elasticsearch/attributes/default.rb

```diff
@@ -1,11 +1,11 @@
 default['elasticsearch'].tap do |elasticsearch|
   # Run Elasticsearch on util instances named elasticsearch_*
   # This is the default
-  elasticsearch['is_elasticsearch_instance'] = ( node['dna']['instance_role'] == 'util' && node['dna']['name'].include?('elasticsearch_') )
+  #elasticsearch['is_elasticsearch_instance'] = ( node['dna']['instance_role'] == 'util' && node['dna']['name'].include?('elasticsearch_') )
 
   # Run Elasticsearch on a solo or app_master instance
   # Not recommended for production environments
-  #elasticsearch['is_elasticsearch_instance'] = ( ['solo', 'app_master'].include?(node['dna']['instance_role']) )
+  elasticsearch['is_elasticsearch_instance'] = ( ['solo', 'app_master'].include?(node['dna']['instance_role']) )
```

Comment:
This change will make ES install by default on an app_master or a solo.  It's not in concordance with the comment on lines 2-3.  Is the default being changed?